src/CoreBundle/private/library/classes/dbobjects/TCMSPortal.class.php

```diff
@@ -192,8 +192,10 @@ public function IsDomainOfThisPortal($sDomainName)
     public function &GetLogo()
     {
         if (!array_key_exists('_portalLogo', $this->_aCache)) {
-            $this->_aCache['_portalLogo'] = new TCMSImage();
-            $this->_aCache['_portalLogo']->Load($this->sqlData['images']);
+            $portalLogo = new TCMSImage();
+            if(false !== $portalLogo->Load($this->sqlData['images'])){
+                $this->_aCache['_portalLogo'] = $portalLogo;
```

Comment:
The value $this->_aCache['_portalLogo'] must be set in any case (in error case null if that works?) because it is used for the return value.

Reply:
okay, changes resetted.